data/src/test/java/org/apache/iceberg/data/DeleteReadTests.java

```diff
@@ -330,6 +407,15 @@ private StructLikeSet rowSetWithoutIds(int... idsToRemove) {
     return set;
   }
 
+  protected StructLikeSet rowSetWithoutIds2(int... idsToRemove) {
```

Comment:
Instead of duplicating logic, please update the existing method so that the table is passed in. If you want to minimize changes, just add an override with the old method signature.